x-pack/plugins/ml/public/application/model_management/expanded_row.tsx

```diff
@@ -191,239 +195,253 @@ export const ExpandedRow: FC<ExpandedRowProps> = ({ item }) => {
+                              />
+                            </h5>
+                          </EuiTitle>
+                          <EuiSpacer size={'m'} />
+                          <EuiDescriptionList
+                            compressed={true}
+                            type="column"
+                            listItems={formatToListItems(stats.model_size_stats)}
+                          />
+                        </EuiPanel>
+                      </EuiFlexItem>
+                    ) : null}
+                  </EuiFlexGrid>
+                </div>
+              ),
+            },
+          ]
+        : []),
+      ...((isPopulatedObject(pipelines) && Object.keys(pipelines).length > 0) || stats.ingest
+        ? [
+            {
+              id: 'pipelines',
+              'data-test-subj': 'mlTrainedModelPipelines',
+              name: (
+                <>
+                  <FormattedMessage
+                    id="xpack.ml.trainedModels.modelsList.expandedRow.pipelinesTabLabel"
+                    defaultMessage="Pipelines"
+                  />
+                  {isPopulatedObject(pipelines) ? (
+                    <EuiNotificationBadge>{Object.keys(pipelines).length}</EuiNotificationBadge>
                   ) : null}
-                </EuiFlexGrid>
-              </div>
-            ),
-          },
-        ]
-      : []),
-    ...((isPopulatedObject(pipelines) && Object.keys(pipelines).length > 0) || stats.ingest
-      ? [
-          {
-            id: 'pipelines',
-            'data-test-subj': 'mlTrainedModelPipelines',
-            name: (
-              <>
-                <FormattedMessage
-                  id="xpack.ml.trainedModels.modelsList.expandedRow.pipelinesTabLabel"
-                  defaultMessage="Pipelines"
-                />
-                {isPopulatedObject(pipelines) ? (
-                  <EuiNotificationBadge>{Object.keys(pipelines).length}</EuiNotificationBadge>
-                ) : null}
-              </>
-            ),
-            content: (
-              <div data-test-subj={'mlTrainedModelPipelinesContent'}>
-                <EuiSpacer size={'s'} />
-                <ModelPipelines pipelines={pipelines!} ingestStats={stats.ingest} />
-              </div>
-            ),
-          },
-        ]
-      : []),
-  ];
+                </>
+              ),
+              content: (
+                <div data-test-subj={'mlTrainedModelPipelinesContent'}>
+                  <EuiSpacer size={'s'} />
+                  <ModelPipelines pipelines={pipelines!} ingestStats={stats.ingest} />
+                </div>
+              ),
+            },
+          ]
+        : []),
+    ];
+  }, [
+    analyticsConfig,
+    deploymentStatItems,
+    details,
+    formatToListItems,
+    inferenceConfig,
+    inferenceStats,
+    pipelines,
+    restMetaData,
+    stats,
+  ]);
+
+  const initialSelectedTab =
+    item.state === 'started' ? tabs.find((t) => t.id === 'stats') : tabs[0];
```

Comment:
```suggestion
    item.state === DEPLOYMENT_STATE.STARTED ? tabs.find((t) => t.id === 'stats') : tabs[0];
```

Reply:
@jgowdyelastic I wonder if it is actually worth an extra import? It's type-safe already and in case we want to rename it one day, it's going to be easy because the value is derived from a type definition.

Reply:
I think the types aren't quite correct for this, when I opened it in an editor `item.state` had an `any` type.
But yeah, if the `state` type can be corrected, not adding the constant to the file makes sense.

Reply:
just checked and the issue is the [MODEL_STATE](https://github.com/elastic/kibana/blob/59b1ba976720e69588ef2abad92b65fa523f6b9a/x-pack/packages/ml/trained_models_utils/src/constants/trained_models.ts#L62) dictionary that contains i18n strings. I guess I need to separate string literals and the actual translation.

Reply:
@jgowdyelastic fixed it by making it a regular string literal. There is no need for translation because other deployment state values come from the ML backend and are not translated at the moment. 

<img width="470" alt="image" src="https://github.com/elastic/kibana/assets/5236598/34e9772d-6423-460b-8a7f-ac026059d7f3">